src/en/installconfig-upgrade.md

```diff
@@ -0,0 +1,45 @@
+Title: Upgrade from MAAS 1.9
+
+# Upgrade from MAAS 1.9
+
+MAAS 2 is only supported on Ubuntu 16.04 LTS (Xenial Xerus). This is because 
+Ubuntu 16.04 defaults to Python 3, whereas previous versions of Ubuntu defaulted
+to Python 2.
+
+MAAS versions 1.9 and 1.9.x will continue to be supported on Ubuntu 14.04 LTS
+(Trusty Tahr) until they reach [end-of-life](https://wiki.ubuntu.com/Releases).
+
+However, it is possible to upgrade from MAAS 1.9.x to MAAS 2.x as part of the
+distribution upgrade from Ubuntu 14.04 (or Ubuntu 15.10) to Ubuntu 16.04. The
+MAAS database and MAAS configuration will be migrated to the latest MAAS
+version as part of the upgrade process.
+
+See the '[Upgrading from Ubuntu 14.04 LTS or
+15.10](https://wiki.ubuntu.com/XenialXerus/ReleaseNotes#Upgrading_from_Ubuntu_14.04_LTS_or_15.10)' 
+section in the release notes for Ubuntu 16.04 LTS (Xenial Xerus) for
+instructions on the upgrade process.
+
+## Troubleshooting
+
+Before you start, make sure you've updated your current Ubuntu series to the
+latest packages. You may also need to add both 'apt' and 'dpkg' packages, if
+not installed already, as these are needed by the upgrade process.
```

Comment:
This paragraph should probably be better placed in the upper section.

Maybe just:
[...] make sure all your packages are updated (`sudo apt update && sudo apt full-upgrade`).

Did you encounter an Ubuntu system that does *not* have the 'apt' and 'dpkg' packages installed? This should never be the case.